email_from_template/send_mail.py

```diff
@@ -272,7 +287,11 @@ def render(component, fail_silently=False):
     if not send_mail:
         return mail
 
-    return mail.send()
+    if on_commit:
+        transaction.on_commit(mail.send)
+        return None
```

Comment:
Ok, I've documented that in 1e36363, but it's a bit awkward given the fact that `None` is falsy. I _think_ this is the right return value, but there's an argument to be made that we should return `True` here. What do you think?